### ISSUE TITLE
Create issues from Cxbx-Reloaded input

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/xbox_kernel_test_suite#1]
In Cxbx-Reloaded, this issue described some details regarding testing : https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues/136

For each remark that deserve it, create a separate issue in this project.
Once that review is done, close this issue.

**Comment by PatrickvL** [comment]
There are more Cxbx-Reloaded issues that contain relevant information, like https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues/39

**Comment by RadWolfie** [comment]
Since we have ability "transfer issue", from GitHub directly, to another repo, let's go ahead transfer any issues to this repo.
